lib/app/utils.dart

```diff
@@ -46,6 +46,10 @@ String formatFileSize(int bytes) {
   return '${(bytes / power).toStringAsFixed(index == 0 ? 0 : 1)} ${units[index]}';
 }
 
+extension PowerfulString on String {
```

Comment:
```suggestion
extension Legen...daryString on String {
```

Reply:
Or, to fix the syntax:
```suggestion
extension LegendaryString on String {
```